policy-staging.json

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
There is no real need for scaling on staging other than to test I'm not blowing it up during these initial deploys.

I thought we could perhaps try a bit of load testing or something and check things scale as expected.

Once we're happy this file and staging related stuff should probably go!